steps/oasys/layer3-assessment/create-assessment.ts

```diff
@@ -307,9 +306,27 @@ export const clickEducationTrainingEmpl = async (page: Page) => {
 }
 
 export const clickRelationships = async (page: Page) => {
+    await expandSectionIfNeeded(page, 'Section 2 to 13', '6 - Relationships')
     await doUntil(
         () => page.locator('a', { hasText: '6 - Relationships' }).click(),
         () => expect(page.locator('#contextleft > h3')).toHaveText('6 - Relationships (Layer 3)'),
         { timeout: 60_000, intervals: [500, 1000, 5000] }
     )
 }
+
+const expandSectionIfNeeded = async (page: Page, sectionText: string, linkText: string) => {
+    const sectionLocator = page.locator('a', { hasText: sectionText })
+    const linkLocator = page.locator('a', { hasText: linkText })
+
+    await doUntil(
+        async () => {
+            if (!(await linkLocator.isVisible())) {
+                await sectionLocator.click()
+            }
+        },
+        async () => {
+            await expect(linkLocator).toBeVisible()
+        },
+        { timeout: 60_000, intervals: [500, 1000, 5000] }
+    )
```

Comment:
Do you need the `doUntil` here?  Would this have the same result:

```suggestion
    if (!(await linkLocator.isVisible())) {
        await sectionLocator.click()
    }
    await expect(linkLocator).toBeVisible()
```

Reply:
I have changed now.